docker/webui/run_openqa.sh

```diff
@@ -30,13 +30,14 @@ function webui() {
 }
 
 function all_together_apache() {
+  mode="prefork -m production"
   # run services
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler &
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets &
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-livehandler &
+  start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler $mode &
+  start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets $mode &
+  start_daemon -u geekotest /usr/share/openqa/script/openqa-livehandler $mode &
   start_daemon -u geekotest /usr/share/openqa/script/openqa gru -m production run &
   apache2ctl start
-  start_daemon -u geekotest /usr/share/openqa/script/openqa prefork -m production --proxy
+  start_daemon -u geekotest /usr/share/openqa/script/openqa $mode --proxy
```

Comment:
It would make sense to simply invoke `openqa-webui-daemon` here. When I remember correctly, @okurz added these scripts specifically so the parameters we pass to the systemd units for production setups can be re-used for containerization. If in question, just do what the systemd files do.

I'd also like to note that the flags you added are problematic. We have code which might be blocking for a short but considerable time so we need `-c 1` (and possibly a higher `-w …`). That speaks even more for sticking to our defaults as I said in the previous paragraph.

```suggestion
  start_daemon -u geekotest /usr/share/openqa/script/openqa-webui-daemon
```

Reply:
Thanks @Martchus for your comments. I'll change all to use these scripts like we do if we are using the load balance mode.

Reply:
Do you think I should add these parameters anyway now that we are using the openqa-webui-daemon script?

Reply:
I'd just make it use the `openqa-webui-daemon` script without further parameters.

Reply:
I second that suggestion :)